vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -650,6 +679,12 @@ The absence of `RestrictedProperties` denotes all properties are accessible usin
         <Property Name="Updatable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
           <Annotation Term="Core.Description" String="Entities can be updated" />
         </Property>
+        <Property Name="Upsertable" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Entities can be upserted" />
+        </Property>
+        <Property Name="CollectionUpdate" Type="Capabilities.CollectionUpdateType" Nullable="false">
+          <Annotation Term="Core.Description" String="Entities can be upserted" />

```

Comment:
Is this the right description?

Reply:
No, is PATCH on collection using delta format - with differentiation "only upsert" and "upsert and delete"